cookiesales.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+//Select the HTML element we will be modifing
+var displayResult = document.getElementById('tableResults');
+
+//Create our Object template
+function StoreLocations (storeName,hourlyMinCustomer,hourlyMaxCustomer,avegCookies) {
+  this.storeName = storeName;
+  this.hourlyMinCustomer = hourlyMinCustomer;
+  this.hourlyMaxCustomer = hourlyMaxCustomer;
+  this.avegCookies = avegCookies;
+  this.hourlyCookieSales = [];
+  this.storeHours = ['6am', '7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm'];
+}
+
+//Create first Object Method for StoreLocations, this generates a random number of customers
+StoreLocations.prototype.forcastRandomCustomerPerHour = function(){
+  return Math.ceil(Math.random() * (this.hourlyMaxCustomer - this.hourlyMinCustomer +1) + this.hourlyMinCustomer);
+};
+
+//create second Object Method to calculate number of cookies sold
+StoreLocations.prototype.genCookieSales = function() {
+  return Math.ceil(Math.random()*(this.avegCookies * this.forcastRandomCustomerPerHour()));
```

Comment:
You don't need to call Math.random again here.  The forcastRandomCustomerPerHour function will generate your random customers per hour while avegCookies is the average amount of cooklies per hour.   If you had a min/max cookies per hour you could have a second function to generate a random number of cookies.